geoserver_init/010_init-gs.js

```diff
@@ -69,7 +69,7 @@ async function initGeoserver() {
   if (proxyBaseUrlChanged) {
     console.info(`Set proxy base url to "${proxyBaseUrl}"`);
   } else {
-    console.info('Setting proxy base url failed.');
+    console.error('Failed setting proxy base url', proxyBaseUrl);
```

Comment:
A whitespace is missing

```suggestion
    console.error('Failed setting proxy base url: ', proxyBaseUrl);
```